### ISSUE TITLE
Nav Tabs - Need to hide icons from assistive tech

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#1239]
CENG_3322

Docs: https://react-magma.cengage.com/version/4.3.0/api/nav-tabs/

Success Criteria: https://www.w3.org/WAI/WCAG21/Understanding/non-text-content

in the nav tab examples with icons, the icons lack an accessible name and are exposed to assistive technology. Because of this, someone who uses a screen reader could come across these and not know what they are for even though they're just decorative in nature.

Add aria-hidden="true" to the span that wraps the SVGs in these links.

**Comment by silvalaura** [comment]
@olehnoskov-cengage is this indirectly addressed by your latest icon change?

**Comment by olehnoskov-cengage** [comment]
> [@olehnoskov-cengage](https://github.com/olehnoskov-cengage) is this indirectly addressed by your latest icon change?

Yes, this bug should be fixed by the latest icon changes.
We'll test it in the scope of [this](https://github.com/cengage/react-magma/issues/1945) ticket.
